Work36

mkzheng972 opened the pull request:
currently seeding 50 faker users

mkzheng972 committed:
worked on post api to workoutlogs

mkzheng972 committed:
added validate to createworkoutlog

mkzheng972 committed:
Merge branch 'master' of https://github.com/FoodieFSA/backup into work36

merging from origin

mkzheng972 committed:
successfully seeded one faker user

mkzheng972 committed:
successfully seeded 50 faker users, added dob and imageurl to table

chaohuichen commented on composer.json:
```diff
@@ -11,14 +11,14 @@
         "php": "^7.3",
         "fideloper/proxy": "^4.2",
         "fruitcake/laravel-cors": "^2.0",
+        "fzaninotto/faker": "^1.9",
```
can you revert this line? I think it should be at the require-dev

mkzheng972 committed:
Update composer.json

moved "fzaninotto/faker": "^1.9", to require-dev
side note: i actually did not even use that library, should i remove it^ ?
